comps/animation/requirements.txt

```diff
@@ -0,0 +1,30 @@
+basicsr==1.4.2
```

Comment:
possible to make sure none of below packages in the BASE docker image from habana?  it will impact our BOM (bill of material) list